server/auth.go

```diff
@@ -217,7 +223,7 @@ func (server *HTTPServer) getAuthTokenHeader(w http.ResponseWriter, r *http.Requ
 	} else {
```

Comment:
shall we convert this `else` into an `else if` and use the `else` to return `false` as the authentication type is unknown?

Reply:
good point, but we have a task in the backlog that I'm planning to do to completely drop the `jwt` part because it's really not used anywhere at all

Reply:
ok!